### PR TITLE
Add email log configuration

### PR DESCRIPTION
**Pull request by samwilson**
This is a copy of Event Metrics' email log config.

Bug: T219233

### CODE REVIEW

**Comment by MusikAnimal** [comment]
I recommend including the URI in the email, since it won't normally include the request payload.

I think this only involves two changes:
* https://github.com/wikimedia/eventmetrics/blob/master/src/AppBundle/Logger/WebRequestProcessor.php
* https://github.com/wikimedia/eventmetrics/blob/master/app/config/services.yml#L58-L59

In the WebRequestProcessor you can add whatever else you want too, such as the logged-in user (though maybe that's invasion of privacy?). See the XTools implementation:
* https://github.com/x-tools/xtools/blob/master/src/AppBundle/Monolog/WebProcessorMonolog.php
* https://github.com/x-tools/xtools/blob/master/config/services.yml#L33-L37

**Comment by samwilson** [comment]
Definitely.

I'm currently trying to see if we can add it to the bundle.

**Comment by samwilson** [comment]
Added to the bundle: https://github.com/wikimedia/ToolforgeBundle/pull/16

**Comment by samwilson** [comment]
Okay, I've updated this with the new bundle version.

File: config/packages/prod/monolog.yaml [hunk 1]
```diff
@@ -2,24 +2,27 @@ monolog:
...
+            path: '%kernel.logs_dir%/foo_%kernel.environment%.log'
+        mailer:
+            type: deduplication
+            handler: swift
+        swift:
+            type: swift_mailer
+            level: critical
+            from_email: '%env(MAILER_FROM_EMAIL)%'
+            to_email: '%env(MAILER_TO_EMAIL)%'
+            subject: '[tools.svgtranslate] Error: %%message%%'
```
Comment (MusikAnimal) [inline config/packages/prod/monolog.yaml]: @samwilson `tools.svgtranslate` :/ I'm wondering if there's a `%kernel.root_dir%` or something like that, which we can probably assume is the name of the application

File: .env.dist [hunk 2]
```diff
@@ -14,6 +14,8 @@ APP_SECRET=92d6f6713081a061210b56723f0389e5
 # Delivery is disabled by default via "null://localhost"
 MAILER_URL=null://localhost
 ###< symfony/swiftmailer-bundle ###
+MAILER_FROM_EMAIL=tools.svgtranslate@tools.wmflabs.org
+MAILER_TO_EMAIL=
```
Comment (MusikAnimal) [inline .env.dist]: `.env` is the new standard so this is probably fine, but I think this means it won't work in Event Metrics right now :(

**Pull request merged**